src/Plugin.php

```diff
@@ -210,6 +217,11 @@ public static function init() {
 
     // Override page title on product attribute pages.
     add_filter('woocommerce_page_title', __NAMESPACE__ . '\ProductAttributePageTitle::woocommerce_page_title');
+
+    add_action(self::CRON_EVENT_ENSURE_BACK_IN_STOCK, __NAMESPACE__ . '\Schema::cron_ensure_back_in_stock');
+    if (!wp_next_scheduled(static::CRON_EVENT_ENSURE_BACK_IN_STOCK)) {
+      wp_schedule_event(strtotime('3am tomorrow'), 'daily', self::CRON_EVENT_ENSURE_BACK_IN_STOCK);
+    }
```

Comment:
1. Let's move this up to after line 171 so that it is grouped with the main back-in-stock functionality, please.

1. Please also move the callback from Schema to WooCommerce, below the existing callback for the back-in-stock functionality.  (Ideally we'd group them into an own ProductBackInStock class, but that's for a future issue.)

Reply:
I moved after the entire check for WooCommerce German Market, since it should not be dependent on that.